nucypher/cli/painting.py

```diff
@@ -382,20 +393,37 @@ def prettify_stake(stake, index: int = None) -> str:
     return pretty
 
 
-def paint_stakes(emitter, stakes):
-    title = "======================================= Active Stakes =========================================\n"
-
+def paint_stakes(emitter, stakes, paint_inactive: bool = False):
     header = f'| ~ | Staker | Worker | # | Value    | Duration     | Enactment          '
     breaky = f'|   | ------ | ------ | - | -------- | ------------ | ----------------------------------------- '
 
-    emitter.echo(title)
-    emitter.echo(header, bold=True)
-    emitter.echo(breaky, bold=True)
-    for index, stake in enumerate(stakes):
-        row = prettify_stake(stake=stake, index=index)
-        row_color = 'yellow' if stake.worker_address == BlockchainInterface.NULL_ADDRESS else 'white'
-        emitter.echo(row, color=row_color)
-    emitter.echo('')  # newline
+    active_stakes = sorted((stake for stake in stakes if stake.is_active),
+                           key=lambda some_stake: some_stake.address_index_ordering_key)
+    if active_stakes:
+        title = "======================================= Active Stakes =========================================\n"
+        emitter.echo(title)
+        emitter.echo(header, bold=True)
+        emitter.echo(breaky, bold=True)
+
+        for index, stake in enumerate(active_stakes):
+            row = prettify_stake(stake=stake, index=index)
+            row_color = 'yellow' if stake.worker_address == BlockchainInterface.NULL_ADDRESS else 'white'
+            emitter.echo(row, color=row_color)
+        emitter.echo('')  # newline
+
+    if paint_inactive:
+        title = "\n====================================== Inactive Stakes ========================================\n"
+        emitter.echo(title)
+        emitter.echo(header, bold=True)
+        emitter.echo(breaky, bold=True)
+
+        for stake in sorted([s for s in stakes if s not in active_stakes],
+                            key=lambda some_stake: some_stake.address_index_ordering_key):
+            row = prettify_stake(stake=stake, index=None)
+            emitter.echo(row, color='red')
+        emitter.echo('')  # newline
+    elif not active_stakes:
+        emitter.echo(f"There are no active stakes\n")
```

Comment:
Just to clarify our logic here - we only log `There are no active stakes` if there are no active stakes **AND** `paint_inactive` is false. 

If there are no active stakes but `paint_inactive` is true, then we don't log?

Reply:
L420 loops over non-active stakes. So if `paint_inactive is True` then it prints any existing non-active stake.

Reply:
Apologies if I'm missing something here.

Is it the case that the `stakes` parameter always has at least one stake in it? If `stakes` is empty, but `paint_inactive` is True, we will print the following:
```
========= Inactive Stakes ============
<---- headers----->
------|-------| ....
```
with no stakes listed, correct? So we show empty `Inactive Stakes` titled section?

Reply:
Ah I see. Well, we have an open issue for this command option, we can rethink this to there.

Reply:
Ah I see. Well, we have an open issue for this command option (https://github.com/nucypher/nucypher/issues/1436), we can rethink this to there.

Reply:
👍